dotcom-rendering/src/lib/getLiveblogAdPositions.ts

```diff
@@ -6,37 +6,43 @@ import {
 /**
  * Decides where ads should be inserted on liveblogs/deadblogs.
  */
-const getLiveblogAdPositions = (
-	blocks: Block[],
-	isMobile: boolean,
-): number[] => {
-	let pxSinceAd = 0;
-	let adCounter = 0;
-	const adPositions = [];
+const getLiveblogAdPositions = (blocks: Block[], isMobile: boolean): number[] =>
+	blocks.reduce<{
+		heightSinceAd: number;
+		adCounter: number;
```

Comment:
Is `adCounter` different to `adPositions.length`?

Reply:
Ah thanks nice spot